### PR TITLE
FeedView [PART3] integration fragment

### PR DESCRIPTION
**Pull request by OstapOpalynskyi-fw**
**What**
Create fragment integration screen

**Why**
ASDK-990

**How**
Create separate activity and fragment which contains `FwVideoFeedView`.

**Video**

https://user-images.githubusercontent.com/101661405/212888024-e0d2b446-bfec-4341-8930-fa2d0574f8ed.mp4

### CODE REVIEW

File: feed_integration/app/src/main/java/tv/fw/feedintegration/fragment/FeedViewFragment.kt [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import tv.fw.common.feed.FeedResource
+import tv.fw.feedintegration.R
+import tv.fw.videofeed.FwVideoFeedView
+import tv.fw.videofeed.baseOptions
+import tv.fw.videofeed.viewOptions
+
+/**
+ * A fragment representing a list of Items.
```
Comment (mohsenoid) [inline feed_integration/app/src/main/java/tv/fw/feedintegration/fragment/FeedViewFragment.kt:15]: ❗Is this kinda leftover comment?
❓If no, what do you mean by "a list of items"?

**Review by mohsenoid — APPROVED** [review]
LGTM, depending on your answer to my question. 😇

File: feed_integration/app/src/main/java/tv/fw/feedintegration/fragment/FeedViewFragment.kt (hunk 1 above)
Comment (OstapOpalynskyi-fw) [reply feed_integration/app/src/main/java/tv/fw/feedintegration/fragment/FeedViewFragment.kt:15]: Yes, this is a leftover

Comment (mohsenoid) [reply feed_integration/app/src/main/java/tv/fw/feedintegration/fragment/FeedViewFragment.kt]: Removed

**Comment by mohsenoid** [comment]
I fixed the ci issue.

**Review by mohsenoid — APPROVED** [review]
👌

**Pull request merged**